### DISCUSSION TITLE
why is my custom theme not working?

### DISCUSSION THREAD

**Discussion by Ryzelz** [TomSchimansky/CustomTkinter#1490] [General]
I'm trying to change the json file but its not working. even though I just copy and pasted. 
i did not touch the code of the json file other than changing the name of the file. 

![image](https://user-images.githubusercontent.com/100330023/233779408-375acdf4-c4bc-4d1b-b785-cadb240aa8a7.png)

**Comment by VasigaranAndAngel** [comment]
I think you have to give the full path of the theme file.

**Comment by Ryzelz** [comment]
> I think you have to give the full path of the theme file.

i did...  and I got an error
![image](https://user-images.githubusercontent.com/100330023/233797403-b842375a-0620-464b-ae65-d262385e99cb.png)


I removed 'C:\'
![image](https://user-images.githubusercontent.com/100330023/233797433-e25f9f40-f42e-41d9-aa7c-a45e6e2be007.png)

still got that error...

**Comment by TomSchimansky** [comment]
Maybe something wrong with the `/`? Try loading the file by yourself and see what happens:
```python
with open(path, "r") as file:
    data = json.load(file)
```
I don't think its a CustomTkinter problem.

**Reply by Ryzelz** [reply]
thank for specifying that its not a customtkinter problem... i search the problem on google and recommend to use raw string literals like this 
`customtkinter.set_default_color_theme(r"C:\Users\ryzel\OneDrive\Desktop\CustomTkinter-master\customtkinter\assets\themes\green.json")`

I just added r on the side of the path file. it works now.  (*/ω＼*)
![image](https://user-images.githubusercontent.com/100330023/233813636-33dba1d7-f669-489a-8599-2cbb401ca4d3.png)

**Comment by avalon60** [comment]
As a side note, I would strongly recommend using Path from [pathlib.](https://medium.com/@ageitgey/python-3-quick-tip-the-easy-way-to-deal-with-file-paths-on-windows-mac-and-linux-11a072b58d5f). When you define a path, Path makes dealing with pathnames more portable across operating systems as well as more intuitive to construct.

Your path has backslashes, as though you are working on Windows and you are using a relative pathname, which might not be a good idea, depending on how your program is operating (and depending on where you launch it from, it may or may not work as expected).
